src/librustc_lint/levels.rs

```diff
@@ -383,6 +386,20 @@ impl<'s> LintLevelsBuilder<'s> {
         BuilderPush { prev, changed: prev != self.cur }
     }
 
+    fn check_gated_lint(&self, name: Symbol, span: Span) {
+        if name.as_str() == "unsafe_op_in_unsafe_fn"
+            && !self.sess.features_untracked().unsafe_block_in_unsafe_fn
```

Comment:
I think we have had some in the past

Reply:
Searching through this repo's code and commits, I have found no reference to "unstable lint". Did I miss something?

Reply:
Well you implemented an unstable lint @LeSeulArtichaut but it could not be found via grepping either. :)  That just means that this lint is guarded by a feature gate. We don't usually do that, but it seems to work, so why not?

Reply:
@nikomatsakis maybe we should make the "internal" (rustc-only) lints unstable like this, too?